test/integration_test.py

```diff
@@ -641,6 +630,59 @@ def worker(_):
                     expect_execution = fetch == first_fetch
                     self.assertEqual(1 if expect_execution else 0, len(execution_ids))
 
+    def __init_manifest_filters(self, catalog: CatalogName) -> JSON:
```

Comment:
Method name is strange. Why two underscores? I can't think anything satisfactorily descriptive that's also reasonably terse, so unless you can think of something better I'd just go with
```suggestion
    def _manifest_filters(self, catalog: CatalogName) -> JSON:
```

Reply:
Also, commit title in unnecessarily detailed. I think "Refactor IT manifest filters" is sufficient. The motivation is obvious from context.